crates/rslint_core/src/groups/errors/constructor_super.rs

```diff
@@ -0,0 +1,92 @@
+use crate::rule_prelude::*;
+use ast::{ClassDecl, ClassElement, Expr, Stmt};
+
+declare_lint! {
+    /**
+    Verify calls of `super()` in constructors (constructor-super)
+
+    The `"extends": "rslint:recommended"` property in a configuration file enables this rule.
```

Comment:
Just delete this line considering at this moment it's not true, additionally, id rather not have this in the description itself since it can be displayed better in the docs by just parsing the tags.

Reply:
Will do!